Gui/opensim/view/src/org/opensim/view/Installer.java

```diff
@@ -149,71 +148,49 @@ public void run() {
     private void restorePrefs()
     {
          String currentVersionStr = NbBundle.getMessage(TheApp.class, "CTL_BuildDate");
-         String savedVersionStr = Preferences.userNodeForPackage(TheApp.class).get("BuildDate", null);
+         String savedVersionStr = TheApp.getCurrentVersionPreferences().get("Internal: BuildDate", null);
```

Comment:
It seems like your convention is to use "Duck: Quack Quack Quack" for user-facing preferences and "Internal.VariableName" for preferences that are hidden from the user (see, e.g., L159 below). So perhaps you intended "Internal.BuildDate" here?

Reply:
🦆 🦆  👍